server/views/departures/new.njk

```diff
@@ -0,0 +1,160 @@
+{% from "govuk/components/date-input/macro.njk" import govukDateInput %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/textarea/macro.njk" import govukTextarea %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+
+{% extends "../partials/layout.njk" %}
+
+{% set pageTitle = applicationName + " - Home" %}
+{% set mainClasses = "app-container govuk-body" %}
+
+{% block content %}
+
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <h1>Log a departure</h1>
+            <form action="/premises/{{premisesId}}/bookings/{{bookingId}}/departures" method="post">
+                <input type="hidden" name="_csrf" value="{{ csrfToken }}"/>
+
+                {{ govukDateInput({
+                    id: "dateTime",
+                    namePrefix: "dateTime",
+                    fieldset: {
+                    legend: {
+                        text: "What is the departure date?",
+                        classes: "govuk-fieldset__legend--m"
+                        }
+                    },
+                    hint: {
+                        text: "For example, 27 3 2007"
+                    }
+                }) }}
+
+                <label class="govuk-label govuk-label--m" for="time">
+                    What is the time of departure?
+                </label>
+                <input type="time" id="dateTime-time" name="dateTime-time" class="govuk-input govuk-input--width-5 govuk-!-margin-bottom-4">
```

Comment:
I think this is fine for now - would be good to add this as a macro, so if we do end up changing the approach, we only have to do it in one place 👍

Reply:
Yeah I think it makes sense to do that next time we use a time input

Reply:
Yeah, probably a bit premature for now